ports/nrf/boards/arduino_nano_33_ble_sense/mpconfigboard.h

```diff
@@ -15,6 +15,11 @@
 
 #define MICROPY_PY_MACHINE_UART         (1)
 #define MICROPY_PY_MACHINE_HW_PWM       (1)
+#define MICROPY_PY_MACHINE_PWM          (1)
+#define MICROPY_PY_MACHINE_PWM_INIT     (1)
+#define MICROPY_PY_MACHINE_PWM_DUTY     (1)
+#define MICROPY_PY_MACHINE_PWM_DUTY_U16_NS (1)
+#define MICROPY_PY_MACHINE_PWM_INCLUDEFILE "ports/nrf/modules/machine/pwm.c"
```

Comment:
I don't think it makes sense to repeat these 5 lines in all mpconfigboard.h files.  Instead, the 4 latter ones (_INIT, _DUTY, _DUTY_U16_NS, _INCLUDEFILE) can all go in the common mpconfigport.h file.  And then `MICROPY_PY_MACHINE_PWM` can either stay here in the board specific config file, or default to enabled but with a `#ifndef ...` guard (so a board can disable it if needed).

Reply:
That's a good suggestion. I prefer to have `MICROPY_PY_MACHINE_PWM` in the board files.